common/src/main/java/org/jboss/pnc/common/monitor/PollingMonitor.java

```diff
@@ -84,6 +84,19 @@ public PollingMonitor() {
         return monitor(condition, checkInterval, timeout, DEFAULT_TIME_UNIT);
```

Comment:
I think we can get rid of like 65:
```
timeout = reader.getIntValueFromPropertyOrDefault(PULLING_MONITOR_TIMEOUT_KEY, DEFAULT_TIMEOUT);
```

Since that'll make it confusing on how to configure the default polling monitor timeout (env var v/s config file). Then on line 84, we can just use the 'DEFAULT_TIMEOUT' as our timeout. WDYT?

Reply:
I think I also added that env var in the README.md. So if we decide to remove it, we should also remove it in the README.md

Reply:
@thescouser89  It makes sense to me Dustin! While looking at the code I was already wondering about that, but should we then add pulling_monitor_check_interval to the env vars as well (and remove it as you suggested for pulling_monitor_timeout)? As it would also be confusing that 2 similar properties are set in different ways? 
The pulling_monitor_threadpool is trickier to be moved to I would leave it where it is.
Thanks!

Reply:
I'm a tiny bit confused. pulling_monitor_check_interval should already be an env var.

Out of topic, but I think I remember now why I did set it as env vars. It was because I couldn't access the module config there and had to find some loophole to configure it.

Reply:
Ok sorry, I created the confusion. So looking more carefully, both pulling_monitor_check_interval and pulling_monitor_timeout are configurable via env vars.  I would leave them as indeed there is no access to the module config file there, and they can be used to override the default values.

However, the PollingMonitor is used not only in the OpenshiftStartedEnvironment, but also in the DefaultBuildCoordinator, so I would not change the default values. What if we leave the chance to override the default values also via config file, so that:
* if PollingMonitor is used as a utility class from the common package, users can use default or override them via env vars
* if OpenShiftStartedEnvironment needs to apply different default values for the timeout or check interval, it can do so without impacting others (e.g. DefaultBuildCoordinator)
Depending on the use case, different checks may be tuned with different timeouts and intervals?

Last thing: it is pOllingmonitor right? Not pUllingmonitor. What if I unify all the naming to and env vars to the correct name? :-)
Thanks!

Reply:
To be honest, I think the env vars would be confusing. I would personally remove it and just make the caller specify a custom check internal and timeout if required with method parameters. We also don't use the env vars anymore in our deployments so it's safe to remove.